tests/src/test/scala/org/apache/openwhisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -1195,4 +1199,24 @@ class ContainerPoolObjectTests extends FlatSpec with Matchers with MockFactory {
     pool = pool - 'first
     ContainerPool.remove(pool, MemoryLimit.STD_MEMORY) shouldBe List('second)
   }
+
+  it should "remove expired in order of expiration" in {
```

Comment:
Can we add one more test case to make sure no more than `prewarmExpirationLimit` containers are removed?